### PR TITLE
minor fixes to SF peer setup docs

### PR DESCRIPTION
**Pull request by heavycrystal**
1. encrypted keypair algo selection was too confusing, simplified and removed weaker choices
2. comment lines properly

### CODE REVIEW

**Comment by iamKunalGupta** [comment]
@heavycrystal this can be landed with https://github.com/PeerDB-io/docs/pull/88

**Review by serprex — APPROVED** [review]

File: connect/snowflake.mdx [hunk 1]
```diff
@@ -93,11 +93,11 @@ begin;
    grant create table on schema identifier($internal_schema) to role identifier($role_name);
 
    -- add a statement like this one for each schema you want to have accessed by peerdb
-   grant usage on schema PUBLIC to role identifier($role_name); -- example
+   -- EXAMPLE: grant usage on schema PUBLIC to role identifier($role_name); -- example
```
Comment (serprex) [inline connect/snowflake.mdx:96]:
```suggestion
   -- EXAMPLE: grant usage on schema PUBLIC to role identifier($role_name);
```

**Comment by iamKunal** [comment]
Should it be AES256 or just AES work?

**Comment by iamKunalGupta** [comment]
Should it be AES256 or just AES works?

**Comment by iamKunalGupta** [comment]
@heavycrystal `AES` does not work

**Pull request merged**